CHANGELOG.md

```diff
@@ -22,7 +22,12 @@ constructor. ([#6240](https://github.com/biocore/scikit-bio/issues/624))
 * Added `Sequence.observed_chars` property for obtaining the set of observed characters in a sequence. ([#1075](https://github.com/biocore/scikit-bio/issues/1075))
 * Added `Sequence.frequencies` method for computing character frequencies in a sequence. ([#1074](https://github.com/biocore/scikit-bio/issues/1074))
 * Added experimental class-method ``Sequence.concat`` which will produce a new sequence from an iterable of existing sequences. Parameters control how positional metadata is propagated during a concatenation.
-* ``skbio.io.format.phylip`` now supports sniffing and reading strict, sequential PHYLIP-formatted files into ``skbio.Alignment`` objects. ([#1006](https://github.com/biocore/scikit-bio/issues/1006))
+* ``skbio.io.format.phylip`` now supports sniffing and reading strict, sequential PHYLIP-formatted files into ``skbio.Alignment`` objects. ([#1006](https://github.com/biocore/scikit-bio/issues/1006)).
+* Addition of function ``skbio.diversity.alpha.alpha_diversity``, which supports applying an alpha diversity metric to multiple samples in one call. 
+
+### Performance enhancements
+* Computation of weighted and unweighted UniFrac on multiple samples through ``skbio.diversity.beta.beta_diversity`` is now hundreds of times faster.
+* Computation of Faith's Phylogenetic Diversity on multiple samples, which can now be done through ``skbio.diversity.alpha.alpha_diversity``, is now tens of times faster than the previous approach which required computing this for each sample through individual calls to ``skbio.diversity.alpha.faith_pd``.
```

Comment:
``alpha_diversity`` and ``beta_diversity`` will go to a new file ``skbio/diversity/_driver.py``, and then be imported from ``skbio.diversity``. Tests move too.